src/account/account_repository_db.cpp

```diff
@@ -64,6 +64,16 @@ bool AccountRepositoryDB::save(const AccountInfo &accInfo) {
 	return successful;
 };
 
+bool AccountRepositoryDB::getCharacterByNameAndAccountId(const uint32_t &id, const std::string &name) {
+	auto result = g_database().storeQuery(fmt::format("SELECT `id` FROM `players` WHERE `account_id` = {} AND `name` = {}", id, g_database().escapeString(name)));
+	if (!result) {
+		g_logger().error("Failed to get character: [{}] from account: [{}]!", name, id);
+		return false;
+	}
+
+	return result->countResults() > 0;
```

Comment:
Wouldn't be better, if checks == 1?

Reply:
> Why checks equals 1? It can be more than one result if there is more than one player in the account.

I guess its because of the name

Reply:
Ah yeah, sorry you guys are right. My bad! I'll update